torch/testing/_internal/common_utils.py

```diff
@@ -116,11 +116,9 @@
 if os.getenv("SLOW_TESTS_FILE", ""):
     with open(os.getenv("SLOW_TESTS_FILE"), 'r') as fp:
         slow_tests_dict = json.load(fp)
-        warnings.warn(f"loaded {len(slow_tests_dict)} slow tests")
```

Comment:
This is nice.  I remember this line is flooding the log in some cases, i.e. distributed tests